### PR TITLE
Added loadout: US_Army_2003

### PR DESCRIPTION
**Pull request by AChesheireCat**

### CODE REVIEW

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp [hunk 1]
```diff
@@ -0,0 +1,544 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_army_2003.hpp:3]: No dependency list

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp [hunk 2]
```diff
@@ -0,0 +1,544 @@
...
+	vest[] = {
+		"CUP_V_B_Interceptor_Rifleman_M81",
+		"CFP_US_pasgt_alice_SquadLeader_ERDL"
+	};
+	backPack[] = {"CUP_B_SLA_Medicbag"};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	magazines[] += {
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+	linkedItems[] += {
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_army_2003.hpp:175]: Conflicts with ftl linkedItems[]

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp [hunk 3]
```diff
@@ -0,0 +1,544 @@
...
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	backpack[] = {"usm_pack_alice_prc119"};
+	backpackItems[] = { FAC_GEAR };
+	linkedItems[] += {
+		"CUP_SOFLAM",
+		"ItemGPS"
+	};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_army_2003.hpp:198]: Conflicts with sl, ftl linkedItems[]

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp [hunk 4]
```diff
@@ -0,0 +1,544 @@
...
+		LIST_4("CUP_200Rnd_TE4_Red_Tracer_556x45_M249"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backPack[] = {"usm_pack_200rnd_556_bandoliers"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_army_2003.hpp:220]: No AR ammo

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp [hunk 5]
```diff
@@ -0,0 +1,544 @@
...
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class mmgac : r {
+	displayName = "MMG Ammo Carrier";
+	backPack[] = {
+		"usm_pack_762x51_bandoliers",
+		"usm_pack_762x51_ammobelts"
+	};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_army_2003.hpp:260]: No MMG ammo

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp [hunk 6]
```diff
@@ -0,0 +1,544 @@
...
+};
+
+class mmgtl : aar {
+	displayName = "MMG Team Leader";
+	backPack[] = {
+		"usm_pack_762x51_bandoliers",
+		"usm_pack_762x51_ammobelts"
+	};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_army_2003.hpp:268]: No MMG ammo

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp [hunk 7]
```diff
@@ -0,0 +1,544 @@
...
+class mattl : car {
+	displayName = "MAT Team Leader";
+	backpack[] = {"usm_pack_alice"};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		"CUP_Javelin_M",
+		"ACE_MapTools"
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_army_2003.hpp:317]: Should probably go in the items[] array.

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp [hunk 8]
```diff
@@ -0,0 +1,544 @@
...
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { "MTR_GEAR" };
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_army_2003.hpp:364]: Macros should not have quotation marks

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp [hunk 9]
```diff
@@ -0,0 +1,544 @@
...
+		LIST_3("DemoCharge_Remote_Mag"),
+		LIST_2("SLAMDirectionalMine_Wire_Mag"),
+		"SatchelCharge_Remote_Mag",
+		"ACE_DefusalKit",
+		"ACE_EntrenchingTool",
+	};
+};
+
+class engm : car {
+	displayName = "Combat Engineer (Mines)";
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_army_2003.hpp:521]: No mine detector?

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp [hunk 10]
```diff
@@ -0,0 +1,544 @@
...
+		LIST_2("APERSTripMine_Wire_Mag"),
+		LIST_2("ACE_FlareTripMine_Mag"),
+		LIST_2("ClaymoreDirectionalMine_Remote_Mag")
+	};
+};
+
+class UAV : car {
+	displayName = "UAV Operator";
+	traits[] = {"UAVHacker"};
+	backpack[] = {"B_UAV_01_backpack_F"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_army_2003.hpp:542]: Remove backpack

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp (hunk 4 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/us_army_2003.hpp:220]: USM (apparently) has a script that runs that fills the USM Bandoliers/Medic Backpacks with the appropriate supplies. Found that out when testing in the editor lol.

Comment (Freddo3000) [reply addons/tmf_loadouts/loadouts/us_army_2003.hpp:237]: #38

File: addons/tmf_loadouts/loadouts/us_army_2003.hpp [hunk 11]
```diff
@@ -0,0 +1,560 @@
...
+		LIST_2("HandGrenade"),
+		LIST_4("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class sp : r {
+	displayName = "Spotter";
+	backPack[] = {"CUP_B_AlicePack_Khaki"};
+	backpackItems[] = {
+			LIST_10("CUP_5Rnd_762x51_M24")
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_army_2003.hpp:426]: One tab too many here

**Pull request merged**